packages/odyssey-react-mui/src/labs/PaginatedTable.tsx

```diff
@@ -194,14 +194,23 @@ const PaginatedTable = <TData extends DefaultMaterialReactTableData>({
 
   const modifiedState = useMemo(
     () => ({
-      ...state,
       pagination: {
         pageIndex: pagination.pageIndex,
         pageSize: pagination.pageSize,
       },
       rowSelection,
+      globalFilter,
+      columnFilters,
+      ...state,
```

Comment:
Do we also want this for `initialState`?

Reply:
Discussed separately, but for posterity: it is already in the preferred order: https://github.com/okta/odyssey/blob/4c482302d8ca74fa8566afcf66989064dd397d8a/packages/odyssey-react-mui/src/labs/PaginatedTable.tsx#L187-L193